core/src/View/Kokkos_ViewTraits.hpp

```diff
@@ -464,6 +527,12 @@ struct ViewTraits {
 
   using size_type = typename MemorySpace::size_type;
 
+  static constexpr bool impl_is_customized =
+      !std::is_same_v<void,
+                      decltype(customize_view_arguments(
+                          Impl::ViewArguments<value_type, array_layout,
+                                              device_type, memory_traits>()))>;
```

Comment:
Why do you need this here?

Reply:
I will need this later to do special shit with special ctors in View for the extra arguments of Sacado. It is gonna be a bit hacky, but at least more readable with the `impl_is_customized`.